### PR TITLE
use new GN-core multi-entry-combiner for organization name

### PR DESCRIPTION
**Pull request by davidblasby**
The diff is a bit more complicated because I edited some already commented out code (from the EC).

The only changes in layout-custom-fields is the addition of this template - <xsl:template mode="mode-iso19139" match="gmd:organisationName" priority="2000">

I made some very minor changes to config-edit because if you don't mark something as multi-lingual, the multilingual sections will be removed by update-fixed-info!

This just converts the gmd:organizationName into a custom HTML fragment.  There is a new multi-entry-combiner directive in GN-core that this directly uses.

There is a full example (config and html fragment) that this produces in MultiEntryCombiner.js - see PR, below).

You need these GN-core PRs (backported to 3.10.x) to use this;
https://github.com/geonetwork/core-geonetwork/pull/4597 (contains the directive)
https://github.com/geonetwork/core-geonetwork/pull/4596 (fixes to the keyword picker so it can be used by the multi-entry-combiner directive)

### CODE REVIEW

**Comment by davidblasby** [comment]
@josegar74 - please review.

DO NOT MERGE until the 2 GN-core PRs are merged and backported to GN 3.10.x!

**Comment by ianwallen** [comment]
Sent more detail review to David.  But the main issue I see at the moment is that this change does not seem to allow organizations other than Government of Canada.  I believe it needs to be modified to allow other 3rd party organizations to be entered.

File: src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl
Comment (ianwallen) [inline src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl]: Typo - I think this should be "thesaurus"

File: src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl
Comment (ianwallen) [inline src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl:199]: Not sure why the word "Organization" is in the name?

File: src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl
Comment (ianwallen) [inline src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl]: Another Typo - I think this should be "thesaurus"

File: src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl
Comment (ianwallen) [inline src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl]: "david place2" not the best example :)
Here are some [examples from Public Services and Procurement Canada](https://www.tpsgc-pwgsc.gc.ca/apropos-about/rgnstnnll-rgnztnal-eng.html)

**Review by ianwallen — CHANGES_REQUESTED** [review]
Found a few typo's and small fixes.

The main issue is that it does not seem to allow for other organizations to be added.

**Comment by davidblasby** [comment]
> Sent more detail review to David. But the main issue I see at the moment is that this change does not seem to allow organizations other than Government of Canada. I believe it needs to be modified to allow other 3rd party organizations to be entered.

Hi, Ian,

If as I said in the email, if we have a thesaurus with a list of organizations, i can very easily change the "fixedValue" ("G.Of.C") to a thesaurus keyword picker.

File: src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl
Comment (davidblasby) [reply src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl:199]: This is a title (heading) that's put on the UI to help users.

File: src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl
Comment (davidblasby) [reply src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl]: ;)

**Comment by davidblasby** [comment]
@ianwallen - see the changes to GN-core
For here, I've fixed "thesaurus" and the example.
In GN-core, i've got two language choices - the UI lang and the data-entry lang.

**Comment by davidblasby** [comment]
I am still working on this, but there's too many open pull requests on different branches.  
I will consolidate and continue this in another PR.

**Pull request merged**
